### PR TITLE
[ADD] Migrated stock_picking_line_sequence to v9.

### PR DESCRIPTION
**Pull request by SerpentCS**

### CODE REVIEW

File: stock_picking_reorder_lines/README.rst [hunk 1]
```diff
@@ -11,14 +13,32 @@ For further information, please visit:
...
 * https://www.odoo.com/forum/help-1
 
+Installation
+============
+
+To install this module, you need to install its dependant module:
+
+ * stock_picking_reorder_lines (https://github.com/OCA/stock-logistics-workflow/)
+
+Configuration
```
Comment (pedrobaeza) [inline stock_picking_reorder_lines/README.rst]: Remove empty section.

File: stock_picking_reorder_lines/README.rst [hunk 2]
```diff
@@ -11,14 +13,32 @@ For further information, please visit:
 
 * https://www.odoo.com/forum/help-1
 
+Installation
+============
+
+To install this module, you need to install its dependant module:
+
+ * stock_picking_reorder_lines (https://github.com/OCA/stock-logistics-workflow/)
```
Comment (pedrobaeza) [inline stock_picking_reorder_lines/README.rst]: This module is stock_picking_reorder_line, so there's no real dependency. Remove this.

File: stock_picking_reorder_lines/__openerp__.py [hunk 3]
```diff
@@ -1,42 +1,26 @@
...
-#
+# © 2015 Eficent Business and IT Consulting Services S.L. -
+# Jordi Ballester Alomar
+# © 2015 Serpent Consulting Services Pvt. Ltd. - Sudhir Arya
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+
 {
     'name': 'Stock picking lines with sequence number',
-    'version': '8.0.0.1.0',
+    'version': '9.0.1.0.1',
```
Comment (pedrobaeza) [inline stock_picking_reorder_lines/__openerp__.py]: This is the first 9.0 version, so 9.0.1.0.0

File: stock_picking_reorder_lines/procurement.py [hunk 4]
```diff
@@ -1,26 +1,13 @@
...
-#    but WITHOUT ANY WARRANTY; without even the implied warranty of
-#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-#    GNU Affero General Public License for more details.
-#
-#    You should have received a copy of the GNU Affero General Public License
-#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
-#
-#
+# © 2015 Eficent Business and IT Consulting Services S.L. -
+# Jordi Ballester Alomar
```
Comment (pedrobaeza) [inline stock_picking_reorder_lines/procurement.py]: Only one line per copyright, so remove this one

File: stock_picking_reorder_lines/stock.py [hunk 5]
```diff
@@ -1,43 +1,15 @@
...
-#    but WITHOUT ANY WARRANTY; without even the implied warranty of
-#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-#    GNU Affero General Public License for more details.
-#
-#    You should have received a copy of the GNU Affero General Public License
-#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
-#
-#
+# © 2015 Eficent Business and IT Consulting Services S.L. -
+# Jordi Ballester Alomar
```
Comment (pedrobaeza) [inline stock_picking_reorder_lines/stock.py]: Only one line per copyright

**Comment by pedrobaeza** [comment]
You should convert YML tests to Python ones, as they are deprecated

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

File: stock_picking_line_sequence/__openerp__.py [hunk 6]
```diff
@@ -13,7 +13,7 @@
               "Serpent CS, "
               "Odoo Community Association (OCA)",
     'website': 'http://www.camptocamp.com',
-    'depends': ['stock', 'sale', 'sale_stock'],
```
Comment (rousseldenis) [inline stock_picking_line_sequence/__openerp__.py]: IMO, prefer explicit dependencies than implicit

**Comment by serpentcs-dev1** [comment]
@jbeficent I have made new changes, could you please review?

File: stock_picking_line_sequence/README.rst [hunk 7]
```diff
@@ -11,14 +13,30 @@ For further information, please visit:
 
 * https://www.odoo.com/forum/help-1
 
+Installation
+============
+
+No specific installation requirements.
```
Comment (LoisRForgeFlow) [inline stock_picking_line_sequence/README.rst]: remove empty section.

File: stock_picking_line_sequence/README.rst [hunk 8]
```diff
@@ -11,14 +13,30 @@ For further information, please visit:
...
+
+Configuration
+=============
+
+No configuration is required.
```
Comment (LoisRForgeFlow) [inline stock_picking_line_sequence/README.rst]: remove empty section.

File: stock_picking_line_sequence/__init__.py [hunk 9]
```diff
@@ -0,0 +1,7 @@
+# -*- coding: utf-8 -*-
+#    Copyright 2015 Camptocamp SA
+# © 2015 Eficent Business and IT Consulting Services S.L.
+# © 2015 Serpent Consulting Services Pvt. Ltd.
```
Comment (LoisRForgeFlow) [inline stock_picking_line_sequence/__init__.py]: why 2015? is this that old?

File: stock_picking_line_sequence/models/stock.py [hunk 10]
```diff
@@ -0,0 +1,69 @@
...
+            current_sequence = 1
+            for line in rec.move_lines:
+                line.write({'sequence': current_sequence})
+                current_sequence += 1
+
+    @api.multi
+    def write(self, values):
+        res = super(StockPicking, self).write(values)
+        for rec in self:
+            rec._reset_sequence()
```
Comment (LoisRForgeFlow) [inline stock_picking_line_sequence/models/stock.py]: IMO this makes no sense, the sequence is required to be kept to compare when invoicing or with the sale order (related lines will have same sequence number). If I change a minor thing in the picking the sequence is reset and the relation is lost.

**Comment by Cedric-Pigeon** [comment]
@SerpentCS Hello, do you plan to achieve this PR ?

**Comment by serpentcs-dev1** [comment]
@Cedric-Pigeon Okay Yes, we will resume working on this.

**Comment by AaronHForgeFlow** [comment]
Hi @darshan-serpent  are you still planning to work on this one?

**Comment by serpentcs-dev1** [comment]
Yes, this was closed because I accidentally diverged the branch and then I tried to reset it to the OCA, which took back all the commits and this PR was closed automatically.

Maybe now I'll send a new one for the same.

**Pull request closed without merging**